libbpfgo.go

```diff
@@ -1426,6 +1427,33 @@ func (p *BPFProg) AttachKretprobe(kp string) (*BPFLink, error) {
 	return doAttachKprobe(p, kp, true)
 }
 
+func (p *BPFProg) AttachNetns(networkNamespacePath string) (*BPFLink, error) {
+	const O_RDONLY int = 00
+	fd, err := syscall.Open(networkNamespacePath, O_RDONLY, 0)
+	if fd < 0 {
+		return nil, fmt.Errorf("failed to open network namespace path %s: %w", networkNamespacePath, err)
+	}
+	link := C.bpf_program__attach_netns(p.prog, C.int(fd))
+	if C.IS_ERR_OR_NULL(unsafe.Pointer(link)) {
+		return nil, errptrError(unsafe.Pointer(link), "failed to attach network namespace on %s to program %s", networkNamespacePath, p.name)
+	}
+
+	// fileName will be used in bpfLink.eventName. eventName follows a format
+	// convention and is used to better identify link types and what they are
+	// linked with in case of errors or similar needs. Having eventName as:
+	// netns-progName-/proc/self/ns/net would look weird so replace it
+	// to be netns-progName-proc-self-ns-net instead.
+	fileName := strings.ReplaceAll(networkNamespacePath, "/", "-")
```

Comment:
I think we need to skip index 0 of `networkNamespacePath`.

```suggestion
	fileName := strings.ReplaceAll(networkNamespacePath[1:], "/", "-")
```